src/containers/Form.js

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { EntityDataModelApi, DataApi } from 'lattice';
+
+import FormView from '../components/FormView';
+
+class Form extends React.Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
```

Comment:
Do you mean extracting a const INITIAL_STATE at the top of this page or on a separate sheet, then referencing it? What benefit will this have? This essentially already is an initial state declaration using ES6.